### PR TITLE
Update the Agent Concepts topics

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description

This PR updates the Concepts topics with a focus on:

* Passive voice
* Spelling
* Semantic line breaks
* Markdown syntax (fix image links, update xrefs etc.)
* Update a few cascading var references
* Update some old/stale links and convert to doc/reference links where possible
* Mode two illustrations from the `assets` directory to the standard media location: https://grafana.com/docs/writers-toolkit/write/image-guidelines/#where-to-store-media-assets

#### PR Checklist

- [ ] CHANGELOG.md updated
- [ ] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: docs/sources/flow/config-language/syntax.md [hunk 1]
```diff
@@ -13,56 +13,51 @@ weight: 200
...
-_Blocks_ are used to configure the {{< param "PRODUCT_ROOT_NAME" >}}'s behavior as well as {{< param "PRODUCT_NAME" >}} components by
-grouping any number of attributes or nested blocks using curly braces.
-Blocks have a _name_, an optional _label_ and a body that contains any number
-of arguments and nested unlabeled blocks.
+You use _Blocks_ to configure the {{< param "PRODUCT_ROOT_NAME" >}}'s behavior as well as {{< param "PRODUCT_NAME" >}}
+components by grouping any number of attributes or nested blocks using curly braces.
+Blocks have a _name_, an optional _label_ and a body that contains any number of arguments and nested unlabeled blocks.
 
-Some blocks support being defined more than once.
+Some blocks support can be defined more than once.
```
Comment (mattdurham) [inline docs/sources/flow/config-language/syntax.md:56]: This verbiage feels off to me. Removing the word `support` or keeping the original verbiage?

Comment (clayton-cornell) [reply docs/sources/flow/config-language/syntax.md:56]: Ahhh... you're spot on. I meant to remove `support`

Comment (clayton-cornell) [inline docs/sources/flow/config-language/syntax.md:56]:
```suggestion
Some blocks can be defined more than once.
```

**Review by mattdurham — APPROVED** [review]
LGTM

**Pull request merged**